static/js/lib/validation.js

```diff
@@ -78,11 +79,13 @@ export const legalAddressValidation = yup.object().shape({
       .string()
       .label("First Name")
       .trim()
+      .matches(NAME_REGEX, "Invalid First Name")
       .required(),
     last_name: yup
       .string()
       .label("Last Name")
       .trim()
+      .matches(NAME_REGEX, "Invalid Last Name")
```

Comment:
Could we provide some feedback as to how the user might fix this so we can save some effort on the user having to contact support for something they could probably troubleshoot themselves?

Reply:
Trying to keep it generic and esthetic I have used ```Name cannot start with a special character, And it cannot contain any character from {/^$#*=[]`%_;<>{}}```. Please let me know if this prompt is good enough.

Reply:
@pdpinch FYI, the Above comment, contains the message that will be shown to users on invalid names, I'll merge this PR if you are also good with the above validation error message.

Reply:
@arslanashraf7 could you lowercase the "And", good to go after that

Reply:
@rhysyngsun Sure, I'll update it.